src/lib/y2packager/clients/inst_repositories_initialization.rb

```diff
@@ -88,7 +88,14 @@ def init_installation_repositories
       # @see https://github.com/yast/yast-packager/blob/7e1a0bbb90823b03c15d92f408036a560dca8aa3/src/modules/Packages.rb#L1876
       # @see https://github.com/yast/yast-packager/blob/fbc396df910e297915f9f785fc460e72e30d1948/src/modules/Packages.rb#L1905
       def adjust_base_product_selection
-        if products.size == 1
+        if forced_base_product
+          log.info("control.xml wants to force the #{forced_base_product.name} product")
+
+          forced_base_product.select
+          discarded_products = products.reject { |p| p == forced_base_product }
+
+          log.info("Ignoring the other products: #{discarded_products.map(&:name).join(", ")}")
```

Comment:
For logged messages simple `inspect` is better IMHO as it better represents the internal value.

```suggestion
          log.info("Ignoring the other products: #{discarded_products.map(&:name).inspect}")
```